src/pages/BiddingPage/BiddingPage.js

```diff
@@ -0,0 +1,462 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function BiddingPage() {
+  const [pageMode, setPageMode] = useState('판매');
+  const [buttonMode, setButtonMode] = useState([false, true]);
+  const [modeChange, setModeChange] = useState(true);
+  const [currentId, setCurrentId] = useState(4);
+  const [value, setValue] = useState('');
+  const today = new Date();
+  const thirtyDaysLater = new Date(today.getTime() + 30 * 24 * 60 * 60 * 1000);
+  const [deadLineDate, setDeadLineDate] = useState(thirtyDaysLater);
+
+  function buttonChange(targetId) {
+    if (targetId === 1) {
+      setButtonMode([true, false]);
+      setModeChange(false);
+    } else {
+      setButtonMode([false, true]);
+      setModeChange(true);
+    }
+  }
+
+  function handleInput(e) {
+    const regex = /^[0-9\b]+$/;
+    let inputValue = e.target.value.replace(/,/g, '');
+    inputValue = Number(inputValue);
+    if (inputValue === '' || regex.test(inputValue)) {
+      setValue(inputValue.toLocaleString());
+    }
+  }
+
+  function handleDeadLine(targetId) {
+    setCurrentId(targetId);
+  }
+
+  function handleDate(targetId) {
+    if (targetId === 1) {
+      const oneDayLater = new Date(today.getTime() + 24 * 60 * 60 * 1000);
+      setDeadLineDate(oneDayLater);
+    } else if (targetId === 2) {
+      const threeDaysLater = new Date(
+        today.getTime() + 3 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(threeDaysLater);
+    } else if (targetId === 3) {
+      const sevenDaysLater = new Date(
+        today.getTime() + 7 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(sevenDaysLater);
+    } else if (targetId === 4) {
+      setDeadLineDate(thirtyDaysLater);
+    } else {
+      const sixtyDaysLater = new Date(
+        today.getTime() + 60 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(sixtyDaysLater);
+    }
+  }
```

Comment:
```suggestion
  const dateType = {
    1: new Date(today.getTime() + 24 * 60 * 60 * 1000),
    2: new Date(today.getTime() + 3 * 24 * 60 * 60 * 1000),
    3: new Date(today.getTime() + 7 * 24 * 60 * 60 * 1000),
    4: new Date(today.getTime() + 30 * 24 * 60 * 60 * 1000),
  };

  function handleDate(targetId) {
    if (dateType[targetId]) {
      setDeadLineDate(dateType[targetId]);
    } else {
      const sixtyDaysLater = new Date(
        today.getTime() + 60 * 24 * 60 * 60 * 1000
      );
      setDeadLineDate(sixtyDaysLater);
    }
  }
```

각각의 구조를 분리해서 적용하는 방법이 가독성에도 좋고 차후 유지보수에도 도움이 될 것 같네요!